e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/AwaitHelper.java

```diff
@@ -9,9 +9,9 @@
 
 public class AwaitHelper {
 
-    protected static final int WAIT_FOR_IN_SECONDS = 10;
+    protected static final int WAIT_FOR_IN_SECONDS = 30;
     protected static final int POLL_INTERVAL_MS = 100;
-    protected static final int POLL_DELAY_MS = 10;
+    protected static final int POLL_DELAY_MS = 5000;
```

Comment:
This change will make every await call take at least 5 seconds. Our tests will be very slow.

I know why you've done this though. The database is not updated to its "final" state right away.

Leave these settings as they were and we will find a better way to write the test.